<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartDialog</class>
 <widget class="QDialog" name="CartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="CartTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>651</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
  <widget class="DeleteButton" name="DeleteFromCart">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="CheckOut" name="CheckoutButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeleteButton</class>
   <extends>QPushButton</extends>
   <header location="global">deletebutton.h</header>
  </customwidget>
  <customwidget>
   <class>CheckOut</class>
   <extends>QPushButton</extends>
   <header location="global">checkout.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
